### ISSUE TITLE
About the setting of minimum-stability

### ISSUE THREAD

**Issue by tkk-sato** [acquia/acquia-cms-starterkit#142]
I want to set the minimum-stability to stable instead of dev during the DownloadModules at build time. Is there any way to do this? The source code is as follows
https://github.com/acquia/acquia-cms-starterkit/blob/develop/src/Helpers/Task/Steps/DownloadModules.php#L61

**Comment by rajeshreeputra** [comment]
Yup, we should set the minimum-stability to stable instead of dev.

**Comment by tkk-sato** [comment]
Thank you. I've forgotten what the original intent was, so I will close this now.
